tox.ini

```diff
@@ -360,7 +360,7 @@ commands_pre =
 
   aiopg: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-dbapi pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-aiopg[test]
 
-  datadog: pip install flaky {toxinidir}/exporter/opentelemetry-exporter-datadog[test]
```

Comment:
Made a bit more sense to me to keep this `flaky` dependency in the `extras_require` of the module instead of here in `tox.ini`.